Documentation/contributing.rst

```diff
@@ -1886,6 +1886,11 @@ If you intent to release a new minor release, see the
 
 #. Announce the release in the ``#general`` channel on Slack
 
+#. Bump the version of Cilium used in the Cilium upgrade tests to use the new release
```

Comment:
Shouldn't this happen around step 11 ? We're making a PR there anyway.

Reply:
Also, should this be repeated in the minor release section? we're using x.y.z versions in the tests, as I recall, but even if we didn't we'd need to change it there too?

Reply:
> should this be repeated in the minor release section?

Step 5 of the minor release process says "Follow the Generic Release Process to release <version>", so this step is transitively part of the minor release section.

>  we're using x.y.z versions in the tests, as I recall, but even if we didn't we'd need to change it there too?

I don't understand what you mean here. Please rephrase - can you provide an example?

Reply:
> Shouldn't this happen around step 11 ? We're making a PR there anyway.

Hm, maybe I should update this to say "in all branches which test upgrade from this version (i.e., v1.1) of Cilium to whatever code is being tested in the branch for which the upgrade tests are being ran?"

What you are suggesting is updating the version for the upgrade test for the very branch which we are pushing a new version of Cilium of (sorry for the awkward phrasing, I couldn't find a better way to state it). For instance, I don't want to update the version of Cilium used in the upgrade tests for the v1.1 branch to be the most recent version of v1.1 that I am generating a release for; I however, would want to update the upgrade tests in the v1.2 branch and in master, which both test upgrade from the latest version of v1.1 to v1.2 and master accordingly. 

I hope that clarifies my intention, it's rather verbose / wordy :/

Reply:
We could have a matrix which describes what branches should be updated, especially since we want to be able to support seamless upgrade from a variety of versions of Cilium; it's crucial that we test this.